<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassMessages</class>
 <widget class="QDialog" name="ClassMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>581</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Message Classes</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>520</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Description</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>Enable or disable
message-classes. Each class
of messages displays
information pertinent to
that class.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_options">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_indications">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Indications</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_debugMemoryUse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Memory Use</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_debugPDU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug PDU</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_debugNAK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug NAK</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_PDUFiledata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PDU Filedata</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_PDURetransmittedFD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PDU Retransmitted FD</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_PDUNonFiledata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PDU Non-Filedata</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_stateAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>State All</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_stateChange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>State Change</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBox_all</tabstop>
  <tabstop>checkBox_indications</tabstop>
  <tabstop>checkBox_debugMemoryUse</tabstop>
  <tabstop>checkBox_debugNAK</tabstop>
  <tabstop>checkBox_debugPDU</tabstop>
  <tabstop>checkBox_PDUFiledata</tabstop>
  <tabstop>checkBox_PDUNonFiledata</tabstop>
  <tabstop>checkBox_PDURetransmittedFD</tabstop>
  <tabstop>checkBox_stateAll</tabstop>
  <tabstop>checkBox_stateChange</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClassMessages</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClassMessages</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
